src/components/render-settings-panel.js

```diff
@@ -1,14 +1,108 @@
+// Copyright (c) 2020 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
 import React, {Component} from 'react';
 import styled, {withTheme} from 'styled-components';
 import {Button, Input} from 'kepler.gl/components/common/styled-components';
 import {Delete} from 'kepler.gl/components/common/icons';
 import ItemSelector from 'kepler.gl/components/common/item-selector/item-selector';
+import {Scene} from 'components/scene'; 
+
+
+import {DeckScene, CameraKeyframes} from '@hubble.gl/core';
+import {easing} from 'popmotion';
+import {DeckAdapter} from 'hubble.gl';
 
 const DEFAULT_BUTTON_HEIGHT = '32px';
 const DEFAULT_BUTTON_WIDTH = '64px';
 const DEFAULT_PADDING = '32px';
 const DEFAULT_ROW_GAP = '16px';
 
+function sceneBuilder(animationLoop) {
+  const data = {};
+  const keyframes = {
+    camera: new CameraKeyframes({
+      timings: [0, 3000],
+      keyframes: [
+        {
+          longitude: 0,
+          latitude: 11,
+          zoom: 2,
+          pitch: 0,
+          bearing: 0
+        },
+        {
+          longitude: 0,
+          latitude: 33,
+          zoom: 3,
+          bearing: 90,
+          pitch: 50
+        }
+      ],
+      easings: [easing.easeInOut]
+    })
+  };
+  animationLoop.timeline.attachAnimation(keyframes.camera);
+
+  // TODO: Figure out how to set up the size 
+  return new DeckScene({
+    animationLoop,
+    keyframes,
+    lengthMs: 5000,
+    data,
+   width: 480,
+   height: 460
+  });
+}
+
+const encoderSettings = {
+  framerate: 30,
+  webm: {
+    quality: 0.8
+  },
+  jpeg: {
+    quality: 0.8
+  },
+  gif: {
+    sampleInterval: 1000
+  }
+};
+
+var _core = require("@hubble.gl/core");
```

Comment:
This may be an accidental line of code `var _core = require("@hubble.gl/core");`